elastic-job-lite-console/src/main/resources/bin/start.bat

```diff
@@ -14,6 +14,7 @@ set PORT=%1
 set CFG_DIR=%~dp0%..
 set CLASSPATH=%CFG_DIR%
 set CLASSPATH=%~dp0..\lib\*;%CLASSPATH%
+set CLASSPATH=$~dp0..\ext-lib\*;%CLASSPATH%
```

Comment:
How about merge `\lib\` and `\ext-lib\` in same line?

Reply:
Ok.